blocks/subscriptions-block/components/usePaymentOptions.jsx

```diff
@@ -1,65 +1,79 @@
 import { useState, useEffect } from "react";
 
-import useSales from "./useSales";
+import { useSales } from "@wpmedia/arc-themes-components";
 
 const STRIPE_PAYMENT_METHOD_ID = 18;
 const PAYPAL_METHOD_ID = 10;
 
-const getPaymentMethodByID = (paymentOptions, paymentMethodTypeID, paymentMethodID) => {
+export const getPaymentMethodByID = (paymentOptions, paymentMethodTypeID, paymentMethodID) => {
+	let minStripeIntentsID;
```

Comment:
This `let` should be a `const` below where it's only assigned once.
![image](https://github.com/WPMedia/arc-themes-blocks/assets/2287238/25ab9592-9afe-408a-8e33-6eb7e2d8b6e1)

Reply:
or better yet, just return the result.